ArcGISRuntimeSDKQt_QMLSamples/Scenes/SyncMapViewSceneView/README.md

```diff
@@ -1,25 +1,36 @@
 # Sync map and scene views
 
-This sample demonstrates how to keep the viewpoints of multiple map or scene views in sync, so that navigating one view immediately updates the other.
+Keep the view points of two views (e.g. MapView and SceneView) synchronized with each other.
 
 ![](screenshot.png)
 
+## Use case
+
+You might need to synchronize GeoView viewpoints if you had two map views in one application - a main map and an inset. An inset map view could display all the layers at their full extent and contain a hollow rectangular graphic that represents the visible extent of the main map view. As you zoom or pan in the main map view, the extent graphic in the inset map would adjust accordingly.
+
 ## How to use the sample
 
-Pan, zoom, and rotate the map or scene view. The other view will update automatically to match your navigation. Note that maps are 2D while scenes are 3D, so the results may not look identical, but the centers and scales will be kept the same.
+Interact with the MapView or SceneView by zooming or panning. The other MapView or SceneView will automatically focus on the same viewpoint.
 
 ## How it works
 
-``Esri::ArcGISRuntime::MapView`` and ``Esri::ArcGISRuntime::SceneView`` have a signal ``viewpointChanged`` which is called each time the viewpoint updates. Inside the slot we get the viewpoint of the sender by calling ``currentViewpoint`` with ``ViewpointType::CenterAndScale``. We then pass that viewpoint into the ``setViewpoint`` of the other view, thus synchronizing all the views.
+1. Wire up both `MapView.viewpointChanged` and `SceneView.viewpointChanged` signals for both geo views.
+2. In each slot, get the current viewpoint from the geo view that is being interacted with and then set the viewpoint of the other geo view to the same value.
+3. Note: The reason for setting the viewpoints in multiple slots is to account for different types of interactions that can occur (ie. single click pan -vs- continuous pan, single click zoom in -vs- mouse scroll wheel zoom, etc.).
 
 ## Relevant API
 
-- ``GeoView``
-- ``GeoView::viewpointChanged``
-- ``GeoView::isNavigating``
-- ``GeoView::currentViewpoint``
-- ``GeoView::setViewpoint``
+* GeoView
+* GetCurrentViewPoint
```

Comment:
I don't believe this is what it is called in QML (no get prefix). please double check